nixpkgs_review/buildenv.py

```diff
@@ -20,11 +20,32 @@ def find_nixpkgs_root() -> Optional[str]:
         prefix.append("..")
 
 
-@dataclass
 class Buildenv:
-    allow_aliases: bool
+    def __init__(self, allow_aliases: bool, extra_nixpkgs_config: str) -> None:
+        if not (
+            extra_nixpkgs_config.startswith("{") and extra_nixpkgs_config.endswith("}")
+        ):
+            raise RuntimeError(
+                "--extra-nixpkgs-config should start with `{` and end with `}`"
```

Comment:
```suggestion
                "--extra-nixpkgs-config must start with `{` and end with `}`"
```
should should not abort the execution